.github/workflows/stage.yml

```diff
@@ -0,0 +1,42 @@
+name: Create Release Branch
+
+# TODO: set outputs? optionally trigger release immediately?
```

Comment:
Is this TODO going to be addressed before merge?

Reply:
Updated comment, but short answer is no. We can do this once we are comfortable with the new process, but for now I'd like to keep the steps manual.